cnf-certification-test/lifecycle/suite.go

```diff
@@ -147,6 +148,14 @@ var _ = ginkgo.Describe(common.LifecycleTestKey, func() {
 	ginkgo.It(testID, ginkgo.Label(testID), func() {
 		testPodPersistentVolumeReclaimPolicy(&env)
 	})
+
+	testID = identifiers.XformToGinkgoItIdentifier(identifiers.TestCPUIsolation)
+	ginkgo.It(testID, ginkgo.Label(testID), func() {
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
+		if provider.CPUIsolationTestingEnabled() {
```

Comment:
We should detect isolated pods individually by checking if request cpu == limit cpu, instead of making it a test. Not all the pods in a namespace will be isolated. The CNF is indicating isolation by making the request and limit equal.

Reply:
Okay I can change it up and detect the isolated pods.

Reply:
Okay per the spec again, the last line of the requirement says: `DOORS Requirement: In order to use Isolated CPUs, specific annotations must be defined in the pod spec` so I switched up my logic to check for those annotations first to distinguish what is/isn't an isolated pod.